From 883bbd3203cd5f24611fc3ca83e361ff18daa7d5 Mon Sep 17 00:00:00 2001
From: Ferdinand Xu <cheng.a.xu@intel.com>
Date: Wed, 3 Jun 2015 18:13:38 -0400
Subject: [PATCH 0684/1164] CDH-42363 : HIVE-10904 Use
 beeline-log4j.properties for migrated CLI
 [beeline-cli Branch] (Chinna Rao Lalam, reviewed
 by Ferdinand Xu)

Change-Id: I27ee7aef32c8f8cf54385b061b7260d9a22816d1
---
 bin/ext/cli.sh |    1 +
 1 file changed, 1 insertion(+)

diff --git a/bin/ext/cli.sh b/bin/ext/cli.sh
index 733705a..ef0e8d0 100644
--- a/bin/ext/cli.sh
+++ b/bin/ext/cli.sh
@@ -21,6 +21,7 @@ updateCli() {
     CLASS=org.apache.hadoop.hive.cli.CliDriver
     JAR=hive-cli-*.jar
   else
+    export HADOOP_CLIENT_OPTS="$HADOOP_CLIENT_OPTS -Dlog4j.configuration=beeline-log4j.properties"
     CLASS=org.apache.hive.beeline.cli.HiveCli
     JAR=hive-beeline-*.jar
   fi
-- 
1.7.9.5

